recipes/libnova/all/conandata.yml

```diff
@@ -0,0 +1,16 @@
+sources:
+  "0.1.16":
+    url: "https://sourceforge.net/code-snapshots/git/l/li/libnova/libnova.git/libnova-libnova-edbf65abe27ef1a2520eb9e839daaf58f15a6941.zip"
```

Comment:
It returns 404 for me: 

```
$ wget -v https://sourceforge.net/code-snapshots/git/l/li/libnova/libnova.git/libnova-libnova-edbf65abe27ef1a2520eb9e839daaf58f15a6941.zip
--2021-08-09 13:27:39--  https://sourceforge.net/code-snapshots/git/l/li/libnova/libnova.git/libnova-libnova-edbf65abe27ef1a2520eb9e839daaf58f15a6941.zip
Loaded CA certificate '/etc/ssl/certs/ca-certificates.crt'
Resolving sourceforge.net (sourceforge.net)... 204.68.111.105
Connecting to sourceforge.net (sourceforge.net)|204.68.111.105|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-08-09 13:27:40 ERROR 404: Not Found.
```

Reply:
Okay, it's a temporary link. I'm to download, but first I needed to access sourceforge page and clock on download button to generate the link. We need to find another way.

Reply:
Why? I mean, it works in the context of the recipe.

Reply:
Also not:

```
libnova/0.1.16: Forced build from source
libnova/0.1.16 (test package): Installing package
Requirements
    libnova/0.1.16 from local cache - Cache
Packages
    libnova/0.1.16:19729b9559f3ae196cad45cb2b97468ccb75dcd1 - Build

Installing (downloading, building) binaries...
libnova/0.1.16: Configuring sources in /home/conan/.conan/data/libnova/0.1.16/_/_/source
ERROR: libnova/0.1.16: Error in source() method, line 47
	destination=self._source_subfolder, strip_root=True)
	NotFoundException: Not found: https://sourceforge.net/code-snapshots/git/l/li/libnova/libnova.git/libnova-libnova-edbf65abe27ef1a2520eb9e839daaf58f15a6941.zip
```

It will work only eventually when that link is available. If go there now, generate the link by clicking on the download snapshot, the Conan create will work too.